Source/FieldSolver/FiniteDifferenceSolver/HybridPICModel/HybridPICModel.cpp

```diff
@@ -552,13 +588,29 @@ void HybridPICModel::FieldPush (
 {
     auto& warpx = WarpX::GetInstance();
 
+    amrex::Real const t_old = warpx.gett_old(0);
+
     // Calculate J = curl x B / mu0 - J_ext
     CalculatePlasmaCurrent(Bfield, eb_update_E);
     // Calculate the E-field from Ohm's law
     HybridPICSolveE(Efield, Jfield, Bfield, rhofield, eb_update_E, true);
     warpx.FillBoundaryE(ng, nodal_sync);
+    warpx.ApplyEfieldBoundary(0, PatchType::fine, t_old);
+
     // Push forward the B-field using Faraday's law
-    amrex::Real const t_old = warpx.gett_old(0);
     warpx.EvolveB(dt, dt_type, t_old);
     warpx.FillBoundaryB(ng, nodal_sync);
+    warpx.ApplyBfieldBoundary(0, PatchType::fine, dt_type, t_old);
```

Comment:
These changes should not be needed since the E-field boundary is applied at
https://github.com/ECP-WarpX/WarpX/blob/2cc4fd2c3ad5be96e1aa5811d72a8e1018d925c4/Source/FieldSolver/FiniteDifferenceSolver/HybridPICModel/HybridPICModel.cpp#L353

And the B-field boundary at 
https://github.com/ECP-WarpX/WarpX/blob/2cc4fd2c3ad5be96e1aa5811d72a8e1018d925c4/Source/FieldSolver/WarpXPushFieldsEM.cpp#L932

Reply:
I will double check this again. I was having some issues with the boundaries, so I will remove and test.